compiler/quilt/tools/command.py

```diff
@@ -212,9 +212,10 @@ def config():
     global _registry_url
     _registry_url = None
 
-def _update_auth(team, refresh_token):
+def _update_auth(team, refresh_token, timeout=None):
     response = requests.post("%s/api/token" % get_registry_url(team), data=dict(
-        refresh_token=refresh_token
+        refresh_token=refresh_token,
+        timeout=timeout
```

Comment:
Wait a minute... You're passing it as a POST parameter. There's no way that can work.